test/validate_v2.ts

```diff
@@ -244,12 +248,23 @@ describe('Printings', () => {
 
   it('printing files have valid copy_quantity attributes', () => {
     printingsByFilename.forEach((printing, file) => {
```

Comment:
here and on 259, file is unused.  

nit: i wish github added a "Please fix" button here instead of a link to the action that found that.